client/src/components/Admin/CountryList/index.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+    <div>
+        <b-card>
+            <b-container>
+                <h1>
+                    <b>ADD COUNTRY</b>
+                </h1>
+                <div class="form mb-3">
+
+                    <b-form-input v-model="country.name" placeholder="Enter the CountryName" required></b-form-input>
+                </div>
+                <div class="form mb-3">
+                    <b-form-input v-model="country.description" placeholder="Description" required></b-form-input>
+                </div>
+                <div class="form mb-3">
+                    <b-form-input v-model="country.countryCode" placeholder="Country Code" required></b-form-input>
+                </div>
+                <b-button variant="outline-primary" size="lg" @click="handleAdd">Add</b-button>
+            </b-container>
+        </b-card>
```

Comment:
make seperate component for adding countries,
use props for handling them
change directory name from `/countryList` to country
add newly created components inside them